www/src/examples/Modal/Focus.js

```diff
@@ -0,0 +1,49 @@
+function Example() {
+  const [show, setShow] = useState(false);
+
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+
+  return (
+    <>
+      <Button variant="primary" onClick={handleShow}>
+        Launch demo modal
+      </Button>
+
+      <Modal show={show} onHide={handleClose}>
+        <Modal.Header closeButton>
+          <Modal.Title>Modal heading</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <Form>
+            <Form.Group className="mb-3" controlId="exampleForm.ControlInput1">
+              <Form.Label>Email address</Form.Label>
+              <Form.Control
+                type="email"
+                placeholder="name@example.com"
+                autoFocus
```

Comment:
`autoFocus` attribute on an input to demo the behavior

Reply:
Sorry, I originally thought that `autoFocus` in `@restart/ui` handled inputs explicitly, but that's not the case.  

I think this should be ok.  @jquense , @golota60 did you want to take a look?

Reply:
seems ok. honestly i don't remember why we needed this style override originally.

Reply:
looks alright to me